src/components/WebexMeeting/README.md

```diff
@@ -0,0 +1,35 @@
+# Webex Interstitial Meeting Component
+
+Webex Interstitial meeting component displays the [Webex Local Media](../WebexLocalMedia) and [Webex Meeting Info](../WebexMeetingInfo) components for a meeting.
+
+<p align="center">
+  <img src="./WebexInterstitialMeeting.png" alt="Default Webex Interstitial Meeting" />
```

Comment:
This picture needs to be a gif, showing the interstitial => in-meeting experience.